### ISSUE TITLE
Add way to calculate bin size and bins for a histogram

### ISSUE THREAD

**Issue by kevindougherty-noaa** [NOAA-EMC/emcpy#37]
There is a need to calculate bin size and bins for a histogram automatically based on the data size. A proposed method could be:

```
binsize = (np.max(data) - np.min(data))/np.sqrt(len(data))
# for omf eval
bins = np.arange(0 - (4*np.std(data), 0 + (4*np.std(data), binsize)

# for other eval types
bins = np.arange(np.mean(data) - (4*np.std(data), np.mean(data) + (4*np.std(data), binsize)
```

I believe this would be best added to `src/emcpy/stats.py`.

**Comment by EdwardSafford-NOAA** [comment]
@kevindougherty-noaa I can take this on if it's still needed.

**Comment by kevindougherty-noaa** [comment]
@EdwardSafford-NOAA That would be great, thank you!

**Comment by CoryMartin-NOAA** [comment]
Closed in #86
